packages/features/portfolio/src/containers/portfolio-overview/portfolio-overview-mobile.tsx

```diff
@@ -0,0 +1,88 @@
+import { DisplayChartMobile } from '@notional-finance/mui';
+import { observer } from 'mobx-react-lite';
+import { useTotalsChart } from './hooks';
+import { usePortfolioNOTETable, useVaultHoldingsTable } from '../../hooks';
+import { Box, styled, useTheme } from '@mui/material';
+import { useAppStore } from '@notional-finance/notionable-hooks';
+import PortfolioHoldingsOverview from './containers/portfolio-holdings-overview';
+import LeverageVaultsOverview from './containers/leverage-vaults-overview';
+import LineChart from '@notional-finance/mui/lib/line-chart/line-chart';
+import moment from 'moment';
+import NOTEHoldingsOverview from './containers/note-holdings-overview';
+import { usePortfolioHoldings } from '../portfolio-holdings/use-portfolio-holdings';
+
+const PortfolioOverviewMobile = () => {
+  const theme = useTheme();
+  const { baseCurrency } = useAppStore();
+  const { portfolioHoldingsData } = usePortfolioHoldings(baseCurrency);
+  const { vaultHoldingsData, showVaultHoldingsTable } = useVaultHoldingsTable();
+  const { noteData } = usePortfolioNOTETable();
+  const { barChartData, barConfig, totalsData } = useTotalsChart(baseCurrency);
+
+  return (
+    <Box>
+      <Container>
+        {barChartData && barConfig && (
+          <DisplayChartMobile
+            chartComponent={{
+              chartHeaderTotalsData: totalsData,
+              id: 'net-worth-area-chart',
+              title: 'Net Worth',
+              hideTopGridLine: false,
+              Component: (
+                <LineChart
+                  data={barChartData.map(
+                    ({
+                      timestamp,
+                      totalNetWorth,
+                      totalAssets,
+                      totalDebts,
+                    }) => ({
+                      date: moment(timestamp * 1000).format('MMM'),
```

Comment:
Can you format the the date using getDateString and the total amounts using this logic here:
https://github.com/notional-finance/notional-monorepo/blob/2620acc68236ec3f5de5aefba97dec5314819a04/packages/common/mui/src/lib/bar-chart/bar-chart-tool-tip/bar-chart-tool-tip.tsx#L62-L67